docs/AggregateIntervals.md

```diff
@@ -0,0 +1,68 @@
+# Using the `aggregateintervals` module
+
+This command allows you to compute statistics for a set of genomic intervals.
+
+**Command line arguments:**
+
+Shortcut | Argument | Description
+:-: | :-: | ---
+`-i` | `--input <input file>` | Path to interval list file
+`-o` | `--output <output path>` | Path to which the output file should be written
+`-c` | `--condition <expression>` | Export expression (see below)
+
+This module aggregates statistics for variants in a set of intervals.  The exposed 
+
+**Namespace in the `aggregateintervals` module:**
+
+Identifier | Description
+:-: | ---
+`interval.contig` | chromosome of interval
+`interval.start` | start position of interval
+`interval.end` | end position of interval
+`global` | global annotation
+`variants` | Variant [aggregable](HailExpressionLanguage.md#aggregables).  Aggregator namespace below.
+
+____
+
+**Namespace within `variants` aggregators:**
+
+Identifier | Description
+:-: | ---
+`v` | Variant
+`va` | Variant annotations
+`global` | Global annotations
```

Comment:
Same reasons it's not in `annotateglobal`.  Nested aggregators seems unnecessarily complex